app/assets/stylesheets/band_show.css.scss

```diff
@@ -0,0 +1,15 @@
+.band-header
+{
+  width: 80%;
+  margin: 3% auto;
+  background: {
+    color: #170563;
+  }
+  text-align: center;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.